### PR TITLE
User context

### PR DESCRIPTION
**Pull request by TomasKrin**

### CODE REVIEW

File: src/contexts/UserContext.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  user: null,
+  setUser: () => {},
+  isLoggedIn: false,
+  handleLogOut: () => {},
+});
+
+const UserProvider = ({ children }: PropsWithChildren) => {
+  const [user, setUser] = useLocalStorage<string | null>("user", null);
+  const isLoggedIn = !!user;
+  const handleLogOut = () => {
```
Comment (rokasandreikenas) [inline src/contexts/UserContext.tsx:22]: tarp virs padek

**Review by rokasandreikenas — APPROVED** [review]
super

File: src/contexts/UserContext.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import { PropsWithChildren, createContext } from "react";
+
+import { useLocalStorage } from "../hooks/localStorage";
+
+const UserContext = createContext<{
+  user: string | null;
+  setUser: any;
```
Comment (rokasandreikenas) [inline src/contexts/UserContext.tsx:7]: setUser: (user: User | null) => void;

File: src/contexts/UserContext.tsx (hunk 1 above)
Comment (TomasKrin) [reply src/contexts/UserContext.tsx:22]: pataisiau

**Pull request merged**